TrafficCapture/KafkaPrinter/src/main/java/org/opensearch/KafkaPrinter.java

```diff
@@ -0,0 +1,55 @@
+package org.opensearch;
+
+
+import java.util.ArrayList;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.errors.WakeupException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Properties;
+
+public class KafkaPrinter {
+    private static final Logger log = LoggerFactory.getLogger(KafkaPrinter.class);
+
+    public static void main(String[] args) {
+        String bootstrapServers = args[1];
+        String groupId = "GroupID";
+        String topic = args[2];
+
+        Properties properties = new Properties();
+        properties.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
+        properties.setProperty("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
+        properties.setProperty("value.deserializer", "org.apache.kafka.common.serialization.ByteArrayDeserializer");
+        properties.setProperty(ConsumerConfig.GROUP_ID_CONFIG, groupId);
+        properties.setProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+
+        KafkaConsumer<String, byte[]> consumer = new KafkaConsumer<>(properties);
+        final Thread mainThread = Thread.currentThread();
+
+        try {
+            consumer.subscribe(Collections.singleton(topic));
+
+            while (true) {
+                ConsumerRecords<String, byte[]> records =
+                        consumer.poll(Duration.ofMillis(100));
+
+                for (ConsumerRecord<String, byte[]> record : records) {
+                    log.info(record.value().toString());
```

Comment:
It seems like we are missing the part where we write this record delimited to stdout